kim/Tiltfile

```diff
@@ -0,0 +1,29 @@
+
+# -*- mode: Python -*-
+
+def kim_build(ref, context, ignore=None, extra_flags=None):
```

Comment:
It'd be nice to add `**kwargs` to pass through to `custom_build` so that you can specify `live_update` etc.